### ISSUE TITLE
Accessibility issues in Xamarin Forms (UWP & iOS platform)

### ISSUE THREAD

**Issue by Sangeetha-Raju** [xamarin/Xamarin.Forms#4859]
Hi Team,

### Description
If I swipe an item either in UWP and iOS platform, touch accessibility doesn't pass to the next time. 

### Steps to Reproduce
1. Run the attached sample.
2. Enable the accessibility in the device.
3. Swipe from right to left on the device.
Issue: Next item is not focused

### Expected Behavior
If I swipe from right to left on the device, the touch should pass to the next item and it should focus. Similarly, if I swipe an item from left to right on the device, the touch should pass to the previous item.

### Actual Behavior
Swipe from right to left on the device, the touch doesn't pass to the next item. So, corresponding item's voice message is not appeared.  

### Note
In Xamarin Forms UWP platform, it is working as expected in v-2.4.0.280 and not working in above versions.
In Xamarin Forms iOS platform, it does not work either in v-2.4.0.280 and above. (We have tested from Xamarin Forms version 2.4.0.280 to till latest.)

### Basic Information

- Version with issue: Xamarin Forms v-2.5.1.527436
- Last known good version: Xamarin Forms v-2.4.0.280
- IDE: Visual Studio 2017 version 15.7.4
- Platform Target Frameworks: 
  - Xamarin.iOS SDK 11.0 
  - UWP:  10240
- Nuget Packages: Xamarin Forms v-2.5.1.527436
- Affected Devices: Windows Simulator.

### Reproduction Link

[Bug.zip](https://github.com/xamarin/Xamarin.Forms/files/2709491/Bug.zip)

**Comment by PureWeen** [comment]
@Sangeetha-Raju can you test with the latest 4.0 pre 1 and let me know if that works for you?
